src/block-components/alignment/editor.scss

```diff
@@ -37,3 +37,7 @@
 		}
 	}
 }
+
+.stk-block-columns .block-editor-block-list__layout {
+	width: 100%;
+}
```

Comment:
What is this for? This will affect a lot of our blocks, I would suggest not having this general selector